CBinding/Parser/DataWrapper.cs

```diff
@@ -30,56 +30,67 @@
 //
 
 using System;
-using System.Collections;
-using System.Collections.Generic;
-
-
-using MonoDevelop.Ide.Gui;
 using MonoDevelop.Ide.CodeCompletion;
-
-using CBinding.Parser;
-using MonoDevelop.Core;
 using MonoDevelop.Ide.Editor;
 using System.Threading.Tasks;
 using System.Threading;
+using System.Text;
+using System.Collections.Generic;
 
-namespace CBinding
+namespace CBinding.Parser
 {
 	sealed class DataWrapper : ParameterHintingData
 	{
-		readonly Function f;
+		OverloadCandidate Overload {
```

Comment:
This can be a one-liner.

Nice use of C# 6 btw, but since it's private IMO a readonly field would be cleaner than a get-only property.